packages/datepicker/src/components/DatePickerInput.tsx

```diff
@@ -21,23 +21,27 @@ type InputProps = {
 };
 
 const defaultStyleProps: StyleProps = {
-  spacing: 'small',
+  spacing: 'medium',
 };
 
 const useStyles = makeStyles<FusionTheme, StyleProps>(
   (theme) =>
     createStyles({
       container: ({ spacing, disabled }) => ({
         ...theme.spacing.comfortable[spacing].style,
+        display: 'flex',
         backgroundColor: theme.colors.ui.background__light.value.hex,
-        borderBottom: disabled ? 'none' : `1px solid ${theme.colors.text.static_icons__tertiary.value.hex}`,
+        boxShadow: disabled ? 'none' : `0px -1px 0px 0px inset ${theme.colors.text.static_icons__tertiary.value.hex}`,
         '&:focus-within': {
-          outline: `2px solid ${theme.colors.interactive.focus.value.hex}`,
-          borderBottom: '1px solid transparent',
+          boxShadow: `0px -2px 0px 0px inset ${theme.colors.interactive.focus.value.hex}`,
         },
-        display: 'flex',
         justifyContent: 'center',
         alignItems: 'center',
+        borderRadius: '0.5em 0.5em 0 0',
```

Comment:
Do we have `theme.borderRadius.medium` type of values instead of using em or rem?